### PR TITLE
Face Mask Detection using ResNet50 v2

### PR DESCRIPTION
**Pull request by RaghavModi**
#### Issue Number
ISSUE #96

#### Describe the changes you've made
Here I have added ResNetv2 50 to detect face mask.

### CODE REVIEW

**Comment by chandrikadeb7** [comment]
@RaghavModi Please make the Readme better like the existing one.

**Comment by RaghavModi** [comment]
Please have a look again I have tried to improve Readme.md. Please suggest if any changes are required.

**Review by chandrikadeb7 — COMMENTED** [review]
> @chandrikadeb7 Please have a look again I have tried to improve Readme.md. Please suggest if any changes are required.

Good! 
Please include a demo gif of video/image input with many faces in a frame in order to highlight the model accuracy.

File: ResNet50_v2/Readme.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Mask Detection using ResNet50 v2 
+
+## Steps to run the ResNet50 v2
+
+1. To clone the Repository: <br>
+git clone https://github.com/chandrikadeb7/Face-Mask-Detection.git
```
[the remark this answers is not in the document]
Comment (RaghavModi) [reply ResNet50_v2/Readme.md]: As suggested I am made the changes. Please review it.

**Comment by chandrikadeb7** [comment]
@RaghavModi Good work!

**Comment by gabrielgiorisatto** [comment]
Do you have the pre-trained model available to download?

**Pull request merged**